ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/user-labels-tags/provision/cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision-chain.yaml

```diff
@@ -0,0 +1,105 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-user-labels-tags-provision
+  steps: 
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-gcp-user-labels-tags
+    - ref: ipi-conf-credentials-mode
+    - ref: gcp-provision-user-tags-serviceaccount
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - ref: optional-operators-subscribe
+    - ref: cucushift-installer-check-gcp-user-labels
+    - ref: cucushift-installer-check-gcp-user-labels-pd-csi
+    - ref: cucushift-installer-check-gcp-user-labels-filestore-csi
+    - chain: cucushift-installer-check
+  env:
+  - name: USER_TAGS
```

Comment:
> Per the definition of userTags and userLables [1], various character combinations should be covered in CI, and let's configure the maximum number of tags: 50.

I will leave this chage to @yunjiang29 to review.

Reply:
@jianli-wei @jianlinliu  the tags and labels lgtm, as these tags and labels are also required in the post step for verification (https://github.com/openshift/release/pull/42819/files#r1361469788), can we consider to write them into shared files, so post step can refer to them easily, e.g.
```
ref:  step-to-write-tags-and-lables-to-shared-files
ref: ipi-conf-gcp-user-labels-tags
ref: post-check-above-tags-and-labels
```

Reply:
Updated the post-installation checks of userLabels, where the expected user-defined labels would be from the env `USER_LABELS`. Thanks!